package/gluon-web-logging/luasrc/lib/gluon/web/model/admin/logging.lua

```diff
@@ -0,0 +1,49 @@
+local uci = require('simple-uci').cursor()
+local system = uci:get_first('system', 'system')
+
+local f = Form(translate('Logging'), translate(
+	'If you want to use a remote syslog server, you can set it up here. '
+	.. 'Please keep in mind, that the data is transmitted unencrypted.'
+))
+local s = f:section(Section)
+
+local enable = s:option(Flag, 'log_remote', translate('Enable'))
+enable.default = uci:get_bool('system', system, 'log_remote')
+enable.optional = true
+function enable:write(data)
+	uci:set('system', system, 'log_remote', data)
+end
+
+local ip = s:option(Value, 'log_ip', translate('IP'))
+ip.default = uci:get('system', system, 'log_ip')
+ip:depends(enable, true)
+ip.optional = false
+ip.placeholder = '0.0.0.0'
+ip.datatype = 'ipaddr'
+function ip:write(data)
+	if data ~= nil then
+		uci:set('system', system, 'log_ip', data)
+	else
+		uci:delete('system', system, 'log_ip')
+	end
+end
+
+local port = s:option(Value, 'log_port', translate('Port'))
+port.default = uci:get('system', system, 'log_port')
+port:depends(enable, true)
+port.optional = true
+port.placeholder = 514
+port.datatype = 'uinteger'
+function port:write(data)
+	if data ~= nil then
```

Comment:
without the if/else here I got some trouble while testing, because the whole section was renamed to log_port if data was nil.

before:
```
system.@system[0]=system
system.@system[0].timezone='CET-1CEST,M3.5.0,M10.5.0/3'
system.@system[0].hostname='xxx'
system.@system[0].log_ip='192.168.1.8'
system.@system[0].log_remote='0'
```

afterwards:
```
system.@log_port[0]=log_port
system.@log_port[0].timezone='CET-1CEST,M3.5.0,M10.5.0/3'
system.@log_port[0].hostname='xxx'
system.@log_port[0].log_ip='192.168.1.8'
system.@log_port[0].log_remote='1'
```

and that seems expactably to me after reviewing: https://github.com/freifunk-gluon/packages/blob/master/libs/lua-simple-uci/src/simple-uci.lua#L34